packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md

```diff
@@ -1,22 +1,32 @@
 
-1. In your Okta org, click **Applications**, and then **Add Application**.
+1. Sign in to your Okta organization with your administrator account.
+1. From the Admin Console side navigation, click **Applications** > **Applications**.
+1. Click **Add Application**.
 
-2. Select the appropriate platform for your use case, enter a name for your new application, and then click **Next**.
+If you want to add an existing OIDC app integration:
 
-3. Add one or more **Login redirect URIs**. This is where the user is directed after they authenticate with the Identity Provider.
+1. Enter the name of the app integration in the **Search...** text box
+1. On the catalog page for the app integration, click **Add**.
+1. Enter a label for your copy of this app integration. Click **Done** to add this to your org.
+1. On the **Assignments** settings tab, click **Assign** to assign the app integration to any user or group in your org. Click **Done** when the assignments are complete.
 
-4. Assign the group of your choosing (if you [set Group Assignments](/docs/reference/social-settings/) for your app) or leave the **Everyone** default.
-
-5. In the **Grant type allowed** section, enable **Implicit**. Using the [Implicit](/docs/guides/implement-implicit/overview/) flow streamlines authentication by returning tokens without introducing any unnecessary additional steps. It allows you to get an ID token quickly, which makes it easy to test your configuration.
+If you need to create a new OIDC app integration:
 
+1. Click **Create New App**.
+1. Select the appropriate platform to match your external application environment and select the **OpenID Connect** sign-on method. Click **Create**.
+1. Enter a name for your new app integration.
+1. Add one or more **Login redirect URIs**. This is where the user is directed after they authenticate with the Identity Provider.
+1. Click **Save**.
+1. Click **Edit** to change the **General Settings** pane. In the **Allowed grant types** section, enable **Implicit**. Using the [Implicit](/docs/guides/implement-implicit/overview/) flow streamlines authentication by returning tokens without introducing additional steps. It allows you to get an ID token quickly, which makes it easy to test your configuration. Click **Save** to confirm your changes.
     > **Note:** The Authorization Code grant flow is also supported.
+1. On the **Assignments** settings tab, click **Assign** to assign the app integration to any user or group in your org. Click **Done** when the assignments are complete.
 
-6. Click **Done**.
+To get the client credentials for your app integration:
 
-7. Scroll to the **Client Credentials** section and copy the client ID that you use to complete the Authorize URL in the next step.
+1. On the **General** settings tab, copy the **Client ID** from the **Client Credentials** section. You need this ID to complete the Authorize URL in the next section.
```

Comment:
why is it **General** settings tab? The name of the tab is the **General** tab.

Reply:
Right I used it in the adjective sense - this is a 'settings tab' - there is a General settings tab, a Sign On settings tab, an Assignments settings tab... I get where you are coming from, and it would clearer if we didn't have "General settings" section on the "General" tab.

Reply:
But putting "settings" before "tab" implies that is part of the NAME of the tab. I think it's confusing.

Reply:
Also fixed "Assignments settings"